src/hybrid.jl

```diff
@@ -126,24 +136,23 @@ function KernelDensity.InterpKDE{IT<:Grid.InterpType}(k::UnivariateKDE, bc::Numb
 end
 
 
-function write_model(model::HybridNB, filename::AbstractString)
+function write_model{S <: AbstractString}(model::HybridNB, filename::S)
     h5open(filename, "w") do f
-        f["NameType"] = "$(eltype(model.kde_names))"
-        info("Writing a model with names of type $(eltype(model.kde_names))")
-        grp = g_create(f, "Classes")
-        grp["Label"] = model.classes
-        grp["Prior"] = collect(values(model.priors))
+        name_type = eltype(keys(model.c_kdes[model.classes[1]]))
```

Comment:
it gets the type of the keys. `model.c_kdes` is for the classes which are integers. we want to go down to a deeper level: `model.c_kdes[0]` is a dictionary as well